app/config/project_map.yml.dist

```diff
@@ -0,0 +1,4 @@
+parameters:
+  # Replace <PROJECT_NAME> and <NEW_PROJECT_NAME> with the appropriate values.
+  # One mapping per project, but if you do not map the tables it uses the given project names
```

Comment:
I'm confused about what this means (oh, and it should say 'projects' rather than 'tables', I think?).

I thought it'd possible to have multiple project names defined on the left hand side, mapping to a actual project names on the right, e.g.:

    app.project.alias1: 'wiki.example.org'
    app.project.alias2: 'wiki.example.org'
    app.project.alias3: 'wiki2.example.org'